public/app/features/library-panels/components/OpenLibraryPanelModal/OpenLibraryPanelModal.tsx

```diff
@@ -31,10 +31,7 @@ export function OpenLibraryPanelModal({ libraryPanel, onDismiss }: OpenLibraryPa
     (searchString: string) => loadOptionsAsync(libraryPanel.uid, searchString, setLoading),
     [libraryPanel.uid]
   );
-  const debouncedLoadOptions = useMemo(
-    () => debounce(loadOptions, 300, { leading: true, trailing: true }),
-    [loadOptions]
-  );
+  const debouncedLoadOptions = useMemo(() => debounce(loadOptions, 300, { leading: true }), [loadOptions]);
```

Comment:
I _think_ we always want `trailing: true` for search, no? Otherwise won't the last letter the user types not be used for search?

I always find debounce/throttle is difficult to wrap my head around 😓